alignak/objects/schedulingitem.py

```diff
@@ -1784,9 +1747,8 @@ def consume_result(self, chk, notif_period, hosts,  # pylint: disable=R0915,R091
                 self.raise_alert_log_entry()
                 self.get_event_handlers(hosts, macromodulations, timeperiods)
 
-        # If no OK in a no OK: if hard, still hard, if soft,
-        # check at self.max_check_attempts
-        # when we go in hard, we send notification
+        # from WARNING/CRITICAL/UNKNOWN/UNREACHABLE/DOWN
+        # to WARNING/CRITICAL/UNKNOWN/UNREACHABLE/DOWN
```

Comment:
yes, from _not ok_ to _not ok_ like you have from ok to ok. In this we manage SOFT/HARD state type and so notifications, eventhander...